### ISSUE TITLE
AnnotationBear: Add examples on how to use it in the docs

### ISSUE THREAD

**Issue by sils1297** [coala-analyzer/coala-bears#774]
@abhsag24 can you do this? difficulty/low area/documentation

**Comment by abhsag24** [comment]
sure, though where should this go?

**Comment by sils1297** [comment]
The Dokumentation comment, doctests!

**Comment by jayvdb** [comment]
Am I right that `AnnotationBear` isnt really a bear.  It is appearing in `coala --show-bears`.

**Comment by sils1297** [comment]
Yeah it isn't intended for users, we should probably open an issue to hide
such bears, maybe give them a flag or so...

**Comment by jayvdb** [comment]
Put them in a different directory also?

**Comment by sils1297** [comment]
Good idea!

**Comment by jayvdb** [comment]
We should note that currently AnnotationBear requires extra non-defaulted parameters like `language`, which will break old `.coafile` as happened in https://github.com/coala/coala-bears/issues/1012

**Comment by shrutishrm512** [comment]
Examples can be added in AnnotationBear.py showing by applying the bear on any file then write errors which are coming(and also writing command with it. So is this correct way to add an example?
